lib/phoenix/endpoint.ex

```diff
@@ -218,14 +218,14 @@ defmodule Phoenix.Endpoint do
       `c:broadcast_from/4`, `c:broadcast_from!/4`, `c:local_broadcast/3`,
       and `c:local_broadcast_from/4`
 
-    * for configuration: `c:start_link/0`, `c:config/2`, and `c:config_change/2`
+    * for configuration: `c:start_link/1`, `c:config/2`, and `c:config_change/2`
```

Comment:
This is the only relevant change. The rest are done by the formatter.

Reply:
Thanks @eksperimental! I don't believe we are formatting the phoenix codebase yet. Would you mind removing the rest of the formatting changes?

Reply:
done